paperqa/contrib/zotero.py

```diff
@@ -216,7 +214,33 @@ def iterate(
             num_remaining = limit - actual_i
 
         self.logger.info("Finished downloading papers. Now creating Docs object.")
+        
+    def _get_collection_id(self, collection_name: str) -> str:
+        """ Get the collection id for a given collection name
 
+        Args:
+            collection_name (str): The name of the collection
+
+        Returns:
+            str: collection id
+        """
+     # specfic collection
+        collections = self.collections()
+        collection_id = ""
+
+        for collection in collections:
+            name = collection["data"]["name"]
+            if name  == collection_name:
+                collection_id = collection["data"]["key"]
+                break
+
+        if collection_id:
+            coll_items = self.collection_items(collection_id)
+            self.logger.info(f"Collection '{collection_name}' found: {len(coll_items)} items")
+            
+        else:
+            self.logger.info(f"Collection '{collection_name}' not found")
```

Comment:
This should be an error I think

Reply:
What error?

Reply:
```suggestion
            raise ValueError(f"Collection '{collection_name}' not found")
```

Reply:
I think both are feasible. Just informing the collection name is not found.

Reply:
But it will hit an infinite loop if it does not error here - as it won't be able to get any items